discovery/spec.md

```diff
@@ -917,108 +931,111 @@ Content-Type: application/json
 
 #### `PUT /services/{id}`
 
-This MUST update the collection of Services in the Discovery Endpoint to
-contain the Service in the request. If the Service identified in the
-request exists it MUST be completely replaced by the Service definition in the
-request. If the Service does not exist then a new Service MUST be created.
+This MUST update, or create, the Service referenced by the specified URL.
```

Comment:
Given that a service has a URL as well, I think this would be clearer as:

> This MUST update, or create, the Service identified by the `{id}` in the path.

(Or "as identified by the `id` in the URL" which is the wording for DELETE.)

Reply:
+1

Reply:
ok done